### ISSUE TITLE
Your personal repos

### ISSUE THREAD

**Issue by SylvanFranklin** [fromtheeast710/watchers#1]
This is a fire idea, I would love to help out. Would it be possible to extend this tool so that it could give you a list of repos that you own with stars? I guess there is overlap but I love something to keep track of my projects.

**Comment by fromtheeast710** [comment]
@SylvanFranklin Yes I have moved to stars instead of watched repos since it's deprecated and a bit clunky to work with. Any help would be great since I'm still a Rust beginner.

Sorry for the late reply, don't know why it took me so long to notice this.
